tests/network/sriov.go

```diff
@@ -157,8 +156,8 @@ var _ = Describe("SRIOV", Serial, decorators.SRIOV, func() {
 
 			buf, err := json.Marshal(metadataStruct)
 			Expect(err).ToNot(HaveOccurred())
-			By("mouting cloudinit iso")
-			Expect(mountGuestDevice(vmi, "config-2")).To(Succeed())
+			By("mounting cloudinit iso")
+			Expect(console.RunCommand(vmi, "sudo mount $(blkid  -L config-2) /mnt/\n", time.Second*120)).To(Succeed())
```

Comment:
Shouldn't it work without `\n`?

Reply:
Also, since subshell is being used (`$(...)`) you need to run it using a shell (sh/bash), for example:
```
sudo sh -c "mount $(blkid -L ...) ..."
```